README.md

```diff
@@ -36,16 +38,27 @@ Red Hat Dependency Analytics only accesses your manifest files to analyze your a
 
 ## Quick start
 
-**Prerequisites**
+### Prerequisites
```

Comment:
This doesn't need to be a level 3 heading, please revert this change.  These are prerequisites of the "Quick start" section, and a H3 heading brakes the flow of the procedure.

`### Prerequisites` --> `**Prerequisites**`